### PR TITLE
Add freetype and toolchain hack to ML build script

### PR DESCRIPTION
**Pull request by vyrnix9582046173**
This PR adds the toolchain hack to the Magic Leap build script and enables freetype. (`skia_use_system_freetype2=true`)

### CODE REVIEW

File: build-ml.sh [hunk 1]
```diff
@@ -1,12 +1,28 @@
...
 
-export MLSDK='/mnt/c/Users/avaer/MagicLeap/mlsdk/v0.16.0'
+export MLSDK='/mnt/c/Users/chris/MagicLeap/mlsdk/v0.16.0'
+export MLSDK_WIN=$(echo "$MLSDK" | sed 's/^\/mnt\/\([a-z]\)\//\1:\\/' | sed 's/\//\\/g')
+
+
+if [ ! -d magicleap-js ]; then
+	  git clone https://github.com/webmixedreality/magicleap-js
+  else
+	    pushd magicleap-js
```
Comment (avaer) [inline build-ml.sh]: Weird spacing.

Comment (vyrnix9582046173) [reply build-ml.sh]: solved in d87e06b

**Review by avaer — APPROVED** [review]
Code looks good, waiting on test results.

**Comment by vyrnix9582046173** [comment]
unfortunately skia_use_system_freetype2 isn't actually useful in this case:
https://github.com/modulesio/skia/blob/master/third_party/freetype2/BUILD.gn#L14-L19

**Pull request closed without merging**